frontend/tests/playwright/product/cross_sell.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Cross-sell test', () => {
+   test.beforeEach(async ({ page }) => {
+      await page.goto('/products/iphone-6s-plus-replacement-battery');
+   });
+
+   test('Current item from cross-sell can be added to cart', async ({ page }) => {
+      const products = await page.getByTestId('cross-sell-item');
+      const count = await products.count();
+
+      let currentProductTitle = null;
+      let currentProductPrice = null;
+      let otherProductTitles = [];
+
+      for (let i = 0; i < count; i++) {
+         const product = products.nth(i);
+
+         const isCurrent = await product.locator('span', { hasText: 'Current item' }).isVisible();
+         if (isCurrent) {
+            currentProductPrice = await product.getByTestId('product-price').textContent();
+            await expect(currentProductPrice).toMatch(/^\$[0-9]+(\.[0-9]{2})/);
```

Comment:
Hmm, I am wondering if we should add a utility function for nabbing a product's price as a `float`? This is in reference to #1217

Reply:
Probably!

Reply:
Noted, I'll add it to the issue